README.md

```diff
@@ -34,6 +33,45 @@ Special notes:
  * Since we cannot distribute Java Richards, a patch is required to derive the
    tracing version (`patches/trace_java_richards.diff`)
 
+## Using the scripts here to analyse an existing benchmark
+
+### Requirements
+
+  * Python 2.7 (the code here is not Python 3.x ready)
+  * bzip2 / buzip2
+  * Python modules: numpy, rpy2
+  * R (e.g. `sudo apt-get install r-base`)
```

Comment:
There's no point giving package names for an arbitrary Linux distribution here: they all call them different things. We can have a separate section or file called something like `README.debian` for that sort of stuff.